.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -40,33 +40,44 @@ This is a checklist for PR authors & reviewers. Please make sure to complete all
 - [ ] I wrote clear testing steps that cover the changes made in this PR
     - [ ] I added steps for local testing in the `Tests` section
     - [ ] I added steps for Staging and/or Production testing in the `QA steps` section
-    - [ ] I added steps to cover failure scenarios (if applicable, i.e. verify an input displays the correct error message if the entered data is not correct)
-    - [ ] I added steps to cover offline scenarios (if applicable, i.e. verify the default avatar icon is displayed if app is offline)
+    - [ ] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
+    - [ ] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
 - [ ] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
-- [ ] I ran the tests & verified they passed on:
+- [ ] I ran the tests on **all platforms** & verified they passed on:
     - [ ] iOS / native
     - [ ] Android / native
     - [ ] iOS / Safari
     - [ ] Android / Chrome
     - [ ] MacOS / Chrome
     - [ ] MacOS / Desktop
-- [ ] I verified there are no console errors related to changes in this PR
+- [ ] I verified there are no console errors (if there’s a console error not related to the PR, report it or open an issue for it to be fixed)
 - [ ] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
-    - [ ] I added comments when the code was not self explanatory
-    - [ ] I put all copy / text shown in the product in all `src/languages/*` files (if applicable)
-    - [ ] I followed proper naming convention for platform-specific files (if applicable)
-    - [ ] I followed style guidelines (in [`Styling.md`](https://github.com/Expensify/App/blob/main/STYLING.md)) for all style edits I made
-    - [ ] I followed the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/STYLE.md#jsdocs))
+     - [ ] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. “toggleReport” and not “onIconClick”)
```

Comment:
I think the indentation is off here

Reply:
ah yeah this is on purpose because this is a sub-bullet point of `I followed proper code patterns`

Reply:
![image](https://user-images.githubusercontent.com/87341702/158677310-fa9c4cae-19b1-4699-87d5-93caccebdd21.png)

Reply:
Ahh I see, thanks for pointing that, fixing it